handlebars/character.hbs

```diff
@@ -129,8 +129,8 @@
 
   <!-- Effects Configuration Tab -->
   <section class="tab" data-tab="effects">
-    <p>This section adds automation effects to imported content. Please read the descriptions carefully. Bugs to <a href="https://discord.gg/CpRtdK6wYq">Discord #auto-effect-bugs channel.</a></p>
-    <button type="button" id="default-effects">Apply Recommended Active Effect (Automation) Settings</button>
+    <p class="notification info">This section adds ActiveEffects to imported content intended only to work with <em>highly automated</em> games. Please read the descriptions carefully. If you are not looking to run a highly automated game that these modules are intended for, it is recommended to disable these settings. Bugs to <a href="https://discord.gg/CpRtdK6wYq">Discord #auto-effect-bugs channel.</a></p>
```

Comment:
This isn't entirely true, as some lower automation effects are applied here if the full suite is not present.

Reply:
It is importing effects that rely on modules.